client/public/locales/en/translation.json

```diff
@@ -172,6 +172,9 @@
     "autoSelectTooltip": "Automatically select this answer based on tags associated with the application(s) or archetype.",
     "autoTagTooltip": "Automatically tag this application or archetype with these tags based on this answer to the questionnaire.",
     "assessmentStakeholderHeader": "Select the stakeholder(s) or stakeholder group(s) associated with this assessment.",
+    "allIssuesEffortTooltip": "This column shows the effort weight for a single issue incident.",
+    "applicationEffortTooltip": "This column shows the total application effort.",
+    "affectedAppEffortTooltip": "This column shows the total calculated effort for all incidents related to this file.",
```

Comment:
Hello @mguetta1, I understand your point that the affectedAppEffortTooltip reflects the same effort count as application effort. However, there are three instances where we want to add a tooltip to explain the relevance of effort: for each place where the total file calculated effort of all incidents for this file, total application effort, and effort weight are calculated.
Therefore, here affected files related to incident files "This column shows the total calculated effort for all incidents related to this file.

Reply:
Hi @kpunwatk, thanks for explaining, but this is not true here that `This column shows the total calculated effort for all incidents related to this file.`
This column shows the **total** application effort, if you open the side drawer you will see the effort and incidents amount for the specific `issue` in each file

The message `This column shows the total calculated effort for all incidents related to this file` is true only in case that only one issue was found during analysis